components/event/EventCard.js

```diff
@@ -76,6 +76,18 @@ export default function EventCard({ event, username }) {
               <ReactMarkdown className="text-sm text-gray-500 py-1 flex-wrap">
                 {event.description}
               </ReactMarkdown>
+              <p className="text-sm text-gray-800 py-1 flex-wrap">
+                {event.isVirtual === true && event.isInPerson === false ? (<span>Remote</span>) : 
+                (<span>
```

Comment:
We can add a condition here in case the user forgot to add `location`

Reply:
Okay, should we show a blank space in this area if he forgot to add the location?

Reply:
yeah that should work